.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Version
+[provide output of the `$ openshift version` command]
```

Comment:
(well, more precisely:  if they have a separate client machine, make sure we get versions from both their client machine and master)